kafka-connect/src/test/java/com/hazelcast/jet/contrib/connect/KafkaConnectRabbitMQIntegrationTest.java

```diff
@@ -0,0 +1,122 @@
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+public class KafkaConnectRabbitMQIntegrationTest extends JetTestSupport {
+
+    public static final int ITEM_COUNT = 10_000;
+
+    @Rule
+    public RabbitMQContainer rabbitMQContainer = new RabbitMQContainer()
+            .withExchange("ex", "direct")
+            .withQueue("test-queue")
+            .withNetwork(Network.newNetwork());
+
+    @Test
+    public void readFromRabbitMqViaConnect() throws Exception {
+        System.setProperty("hazelcast.logging.type", "log4j");
+        ConnectionFactory factory = new ConnectionFactory();
+        factory.setUsername(rabbitMQContainer.getAdminUsername());
+        factory.setPassword(rabbitMQContainer.getAdminPassword());
+        factory.setHost(rabbitMQContainer.getContainerIpAddress());
+        factory.setPort(rabbitMQContainer.getAmqpPort());
+
+        Connection conn = factory.newConnection();
+        Channel channel = conn.createChannel();
+        channel.queueBind("test-queue", "ex", "test");
+        byte[] messageBodyBytes = "Hello, world!".getBytes(StandardCharsets.UTF_8);
+
+        publish(channel, messageBodyBytes, ITEM_COUNT);
+
+        Properties rabbitMqProperties = new Properties();
+        rabbitMqProperties.setProperty("name", "rabbitmq-source-connector");
+        rabbitMqProperties.setProperty("connector.class", "com.github.jcustenborder" +
+                ".kafka.connect.rabbitmq.RabbitMQSourceConnector");
+        rabbitMqProperties.setProperty("kafka.topic", "messages");
+        rabbitMqProperties.setProperty("rabbitmq.queue", "test-queue");
+        rabbitMqProperties.setProperty("rabbitmq.host", rabbitMQContainer.getContainerIpAddress());
+        rabbitMqProperties.setProperty("rabbitmq.port", rabbitMQContainer.getAmqpPort().toString());
+        rabbitMqProperties.setProperty("rabbitmq.username", rabbitMQContainer.getAdminUsername());
+        rabbitMqProperties.setProperty("rabbitmq.password", rabbitMQContainer.getAdminPassword());
+
+        Pipeline pipeline = Pipeline.create();
+        pipeline.readFrom(KafkaConnectSources.connect(rabbitMqProperties))
+                .withoutTimestamps()
+                .map(record -> Values.convertToString(record.valueSchema(), record.value()))
+                .writeTo(AssertionSinks.assertCollectedEventually(60,
+                        list -> assertEquals(ITEM_COUNT, list.size())));
+
+        JobConfig jobConfig = new JobConfig();
+        jobConfig.addJarsInZip(Objects.requireNonNull(this.getClass()
+                                                          .getClassLoader()
+                                                          .getResource("kafka-connect-rabbitmq-0.0.2-SNAPSHOT.zip"))
+                                      .getPath()
+        );
+
+        Job job = createJetMember().newJob(pipeline, jobConfig);
+
+        sleepAtLeastSeconds(5);
+        publish(channel, messageBodyBytes, 1);
```

Comment:
we publish 1 extra message after some time, why ?
the assertion sink expects 10K items but we put 10K + 1 how does the test pass ?